client/me/help/help-contact-closed/index.jsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormSectionHeading from 'components/forms/form-section-heading';
+import HelpContactClosedDetail from './detail';
+
+export default localize( ( props ) => {
+	const {	translate } = props;
+
+	return (
+		<div className="help-contact-closed">
+			<FormSectionHeading>{ translate( "We're not available this week" ) }</FormSectionHeading>
+			{ translate(
+				'Private email support is closed at the moment while we work on a few ' +
+				'projects internally as a team. We will reopen on {{strong}}%(date)s{{/strong}}.', {
+					args: {
+						date: 'Thursday, September 22, 2016',
+					},
+					components: {
+						strong: <strong />,
+					}
+				}
+			) }
+			<HelpContactClosedDetail icon="help">
+				{ translate(
+					'The {{link}}forums{{/link}} remain open and staffed during this time, ' +
+					'and you can post a support request using the form below.', {
+						components: {
+							link: <a href="https://en.forums.wordpress.com/" target="_blank" rel="noopener noreferrer" />
+						}
+					}
+				) }
+			</HelpContactClosedDetail>
+			<HelpContactClosedDetail icon="credit-card">
+				{ translate(
+					'If you require a refund, you can request one directly from your {{link}}Manage Purchases{{/link}} ' +
+					'screen.', {
+						components: {
+							link: <a href="/purchases" />
+						}
+					}
+				) }
+			</HelpContactClosedDetail>
+			<HelpContactClosedDetail icon="book">
+				{ translate(
+					'If you are new to WordPress.com we have a step-by-step {{guide_link}}guide{{/guide_link}} to all things WordPress. ' +
+					'You can find more details in our {{support_doc_link}}support documentation{{/support_doc_link}}. There we have ' +
+					'guides on {{get_started_link}}getting started{{/get_started_link}}, {{first_post_link}}writing your first ' +
+					'post{{/first_post_link}}, and {{find_readers_link}}finding your readers{{/find_readers_link}}', {
+						components: {
+							guide_link: <a href="#WHERE-SHOULD-THIS-LINK-GO????" />,
+							support_doc_link: <a href="https://support.wordpress.com/" target="_blank" rel="noopener noreferrer" />,
+							get_started_link: <a href="https://support.wordpress.com/start" target="_blank" rel="noopener noreferrer" />,
+							first_post_link: <a href="https://support.wordpress.com/create" target="_blank" rel="noopener noreferrer" />,
+							find_readers_link: <a href="https://support.wordpress.com/connect" target="_blank" rel="noopener noreferrer" />
```

Comment:
In the original text `find_readers_link` took you to https://en.support.wordpress.com/getting-more-views-and-traffic/, but this one works better! 👍